### DISCUSSION TITLE
Display the number of stars for a repository

### DISCUSSION THREAD

**Discussion by knanao** [rust-lang/crates.io#5300] [Feature Requests]
It might be helpful if the detail page or the list page on `crate.io` would have the number of stars for a repository to compare these crates as one indicator.

Thank you for your consideration on the matter.

**Comment by ToBinio** [comment]
just a general question:

is there any way we could use the Github-api for such info or similar like #5928?
the problem I found is that unauthorized API calls have a rate limit of only 60 requests per hour. Which is most definitely not sufficient.

On the other hand request from an authorized user (uses an API token) has a rate limit of around 5000 / hour. But using a private api-token is also not really possible...
